app/controllers/admin/cms/transcripts_controller.rb

```diff
@@ -2,7 +2,9 @@ class Admin::Cms::TranscriptsController < Admin::ApplicationController
   before_action :set_transcript, only: [:edit, :update]
 
   def new
-    @transcript = Transcript.new(collection_id: collection_id)
+    @transcript = Transcript.new(
+      collection_id: Collection.find_by(uid: params[:collection_id]).id
```

Comment:
i'm confused, finding the collection where the uid == the id?

if you already have the ID in `params[:collection_id]`, then won't `Transcript.new(collection_id: params[:collection_id])` suffice?